book/tut01_firstfigure.ipynb

```diff
@@ -13,20 +13,19 @@
    "source": [
     "# PyGMT - first figure and subplot \n",
     "\n",
-    "#### Preworkshop in 2024 AGU Fall Meeting in Washington, DC, United States\n",
-    "##### Jing-Hui Tong\n",
-    "###### Reference:\n",
-    "###### Liam Toney, [2021 IRIS ROSES unit05](https://github.com/roseseismo/roses2021/blob/main/unit05/roses_mapping_lecture.ipynb)\n",
-    "###### Chih-Ming Lin, GMT courses for 2019 NTNUES summer program\n",
-    "---\n",
     "This tutorial will cover the fundamental concepts behind making figures with PyGMT: \n",
     "- drawing coastlines\n",
     "- drawing a map frame\n",
     "- choosing a projection\n",
     "- downloading remote dataset\n",
     "- imaging Earth's relief\n",
     "- creating colorbar\n",
-    "- subplot in a figure\n"
+    "- subplot in a figure\n",
+    "\n",
+    "---\n",
+    "###### Reference:\n",
+    "###### Liam Toney, [2021 IRIS ROSES unit05](https://github.com/roseseismo/roses2021/blob/main/unit05/roses_mapping_lecture.ipynb)\n",
+    "###### Chih-Ming Lin, GMT courses for 2019 NTNUES summer program"
```

Comment:
These lines look quite big on the website:

![image](https://github.com/user-attachments/assets/5bbef107-961e-4e22-a922-68870d6956c4)

Maybe change to just normal text without the H6 (######) heading:

```suggestion
    "References:\n",
    "- Liam Toney, [2021 IRIS ROSES unit05](https://github.com/roseseismo/roses2021/blob/main/unit05/roses_mapping_lecture.ipynb)\n",
    "- Chih-Ming Lin, GMT courses for 2019 NTNUES summer program"
```

Which will make it look smaller

![image](https://github.com/user-attachments/assets/176c75b8-ac03-42bf-878b-a1f53ec45066)